conf/machine/raspberrypi3-64.conf

```diff
@@ -2,7 +2,7 @@
 #@NAME: RaspberryPi 3 Development Board
 #@DESCRIPTION: Machine configuration for the RaspberryPi 3 in 64 bits mode
 
-MACHINEOVERRIDES = "raspberrypi3:${MACHINE}"
+MACHINEOVERRIDES = "raspberrypi3:raspberrypi3-64:${MACHINE}"
```

Comment:
I left `raspberrypi3` because it's always been that way, and I figured `raspberrypi3-64` was just a refinement.  We can switch to `raspberrypi3-64` if you prefer (it does seem cleaner).  `bluez5` will have to be patched to get the right firmware then.

Reply:
No. What I'm asking is why did you add `raspberrypi3-64`? We already have the MACHINE in MACHINEOVERRIDES.

Reply:
Because I built something wrong and was convinced the raspberrypi3-64 appends to config.txt weren't working.  Rework incoming after retest.